client/galaxy/style/less/reports.less

```diff
@@ -207,3 +213,8 @@
 .quarter_width {
     width: 25%;
 }
+
+.toolFormBody {
```

Comment:
this broke the alignment of all middle panes in galaxy/TS
ping @Airistotal 
![screenshot 2015-08-27 14 49 08](https://cloud.githubusercontent.com/assets/1814954/9529895/d8acc5ac-4cca-11e5-9584-eb1f02655160.png)